src/models/Request.js

```diff
@@ -0,0 +1,15 @@
+export default (sequelize, DataTypes) => {
+  const Request = sequelize.define('requests', {
+    status: {
+      type: DataTypes.ENUM('pending', 'rejected', 'approved'),
+      allowNull: false,
+      defaultValue: 'pending',
+    },
+  }, {});
```

Comment:
Where are the references to the associations you defined for this model?

Reply:
They are in the other models. Sequelize allows you to use has or belong. It's a semantic meaning if you go with has the references are defined in the parent model if it is belonng they are defined in the child

Reply:
The request table should be the one to hold the reference to the `trip`, `user` and `accommodation`. Hence we need to have the foreign keys in the request table.